### DISCUSSION TITLE
RSS feed

### DISCUSSION THREAD

**Discussion by rakema01** [torrust/torrust-index-archive#98] [Feature requests]
As with any tracker, a RSS feed would be nice to be able to have an easier time grabbing new torrents and seeding them. A separate API key for the feed that's bound to a user and has a long expiration date by default (2mo-6mo) is preferable so as to allow users to just set and forget until the key expires.

**Comment by da2ce7** [comment]
- https://github.com/torrust/torrust-index-frontend/discussions/288
